routes/users.js

```diff
@@ -1,9 +1,60 @@
 var express = require('express');
 var router = express.Router();
+var passport = require('passport');
+const db = require('../models');
+
+//This file needs to run and have nothing to export
+require('../controllers/user.controller');
 
 /* GET users listing. */
 router.get('/', function (req, res, next) {
     res.send('respond with a resource');
 });
 
+router.get('/auth/facebook', passport.authenticate('facebook', { scope : ['email']}));
+router.get('/auth/facebook/check', (req,res) => {res.send('everything is fine')});
+
+router.get('/auth/facebook/callback',
+    passport.authenticate('facebook', {
+      successRedirect: 'success',
+      failureRedirect: 'fail'
+    }));
+
+router.get('/auth/facebook/fail', (req,res) => {
+  // Failed login landing page
+  res.send("Something went wrong...");
+});
+
+router.get('/auth/facebook/success', (req,res) => {
+  // Successfull login landing page
+  // res.redirect('/');
+    res.redirect('../../demo');
```

Comment:
as discussed, the FE will be taking responsibility for the redirections, you should just return a response for each situation (i.e 200, 400. etc...)

Reply:
I added it but it is commented out because it will screw up the demo and people still need the demo working